labs/agora.md

```diff
@@ -0,0 +1,62 @@
+It falls upon the end user to conduct their own due diligence and collaborate with our lab maintainers to enhance and deliver improved products to the market.
+
+# Scope of Lab
+_Enter the scope of the lab. This should include enough information to allow the Lab stewards to understand how it fits within Hyperledger's mission._
+
+The lab's scope encompasses the development and nurturing of cryptography libraries and programs, fostering a secure and collaborative space for these endeavors. Our mission aligns with the Greek agora's spirit, providing a platform for the creation, refinement, and dissemination of cryptographic protocols. These protocols are intended not only for experimentation but also for integration into real-world applications. While our primary focus is on facilitating the production use of cryptography, we also welcome research-oriented contributions.
+
+Within this scope, the lab's purpose extends to include existing libraries that are already in use within production systems. 
+By hosting them in this environment, it ensures their continued existence, offering stability and proper licensing and management. 
+
+These libraries help further the Hyperledger mission by facilitating cryptography that is friendly to blockchains.
+Many blockchains use their own cryptography implementations which generally are basic and not suited for decentralized
+settings. Algorithms like distributed key generation, threshold signing and decryption are offered. The lab is not
+necessarily restricted to the blockchain setting as they can function anywhere also.
+
+One last goal of the lab is to have the Linux Foundation own the code i.e. it cannot be claimed by any single entity or
+corporation as their own. They may fork and maintain their own copies and derivative work but the original source
+is retained here.
+
+# Initial Committers
+_Enter the Github IDs for the set of initial committers._
+- https://github.com/mikelodder7
+
+# Sponsor
+_Provide the name of your sponsor. A sponsor is optional, but the sponsor must be a maintainer of one of the Hyperledger projects, a TOC member, or a WG chair. Read about sponsors' duty in [Section 3, Labs proposal](./index.md#process-to-propose-a-new-lab)._
+- Stephen Curran - https://github.com/swcurran
+
+# Pre-existing repository
+_If you currently have a Github repository that you wish to transfer to the Hyperledger Labs organization, please provide a link here. **NOTE: Please refer to the README for additional information on existing repositories.**_
+- https://github.com/mikelodder7/blsful
+- https://github.com/mikelodder7/gennaro-dkg
+- https://github.com/mikelodder7/unknown_order
+- https://github.com/mikelodder7/soteria-rs
+- https://github.com/mikelodder7/vsss-rs
+- https://github.com/mikelodder7/curvey
+- https://github.com/mikelodder7/blstrs
+- https://github.com/mikelodder7/bls12_381_plus
```

Comment:
This repo is forked from zkcrypto. Can you let us know how these two repos relate and if there is anything specific that we need to be aware of?

Reply:
This version offers many additional features: Hash2Curve, Serialization, Formatting, Sum of Products, big and little endian byte IO.


I have submitted a PR to zkcrypto but they have told me it may be years before they can consider it.